infra/core/managed-cluster/.bicep/contributor.bicep

```diff
@@ -0,0 +1,14 @@
+targetScope = 'subscription'
```

Comment:
if we have 2 modules here, it suggests they can be re-used. It might be worth making more 'defaulted' params, such as the targetScope above and any others. they can be defaulted as needed. Else it'll make sense 2 either have 1 module for AKS-specific RBAC or if it split, add flexibility to be used elsewhere?

Reply:
@kbabigumira  @aadev1 @aadev1 @ppotturi

Reply:
Agreed. One generic RBAC module that can be reused in the main aks cluster bicep module

Reply:
Agree - I'll make the changes.